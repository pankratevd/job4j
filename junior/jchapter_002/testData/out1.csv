Всего групп: 1
Группа 1
"111";"123";"222"
"300";"";"100"
"200";"123";"100"

