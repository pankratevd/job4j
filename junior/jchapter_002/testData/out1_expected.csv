Всего групп: 1
Группа 1
"111";"123";"222"
"200";"123";"100"
"300";"";"100"

